### ISSUE TITLE
Using conditions to hide phone number throws an error

### ISSUE THREAD

**Issue by furioursus** [verbb/formie#491]
**Description**
When adding a phone number as requied in a form and hiding it behind the value set inside a dropdown, if that conditional is not met, the form rejects with a 500 error in the AJAX submission notification.

For sanity's sake, I confirmed this was the behavior by removing the phone field's required status, which worked, as well as choosing the dropdown option that would show the phone number field and filling it out there while it _was_ required.

Here's the stack trace and error that gets pushed to `web.log` on error:

```
2021-06-12 12:28:59 [-][1][-][error][TypeError] TypeError: Argument 1 passed to craft\helpers\StringHelper::isBlank() must be of the type string, null given, called in /Users/ckennedy/Code/manetu/vendor/verbb/formie/src/fields/formfields/Phone.php on line 239 and defined in /Users/ckennedy/Code/manetu/vendor/craftcms/cms/src/helpers/StringHelper.php:725
Stack trace:
#0 /Users/ckennedy/Code/manetu/vendor/verbb/formie/src/fields/formfields/Phone.php(239): craft\helpers\StringHelper::isBlank(NULL)
#1 /Users/ckennedy/Code/manetu/vendor/craftcms/cms/src/base/Element.php(1852): verbb\formie\fields\formfields\Phone->validateRequiredFields(Object(verbb\formie\elements\Submission), NULL)
#2 /Users/ckennedy/Code/manetu/vendor/yiisoft/yii2/validators/InlineValidator.php(84): craft\base\Element->validateCustomFieldAttribute('field:phoneNumb...', Array, Object(yii\validators\InlineValidator), Object(verbb\formie\models\Phone))
#3 /Users/ckennedy/Code/manetu/vendor/yiisoft/yii2/validators/Validator.php(261): yii\validators\InlineValidator->validateAttribute(Object(verbb\formie\elements\Submission), 'field:phoneNumb...')
#4 /Users/ckennedy/Code/manetu/vendor/yiisoft/yii2/base/Model.php(368): yii\validators\Validator->validateAttributes(Object(verbb\formie\elements\Submission), Array)
#5 /Users/ckennedy/Code/manetu/vendor/verbb/formie/src/elements/Submission.php(396): yii\base\Model->validate(Array, true)
#6 /Users/ckennedy/Code/manetu/vendor/verbb/formie/src/controllers/SubmissionsController.php(457): verbb\formie\elements\Submission->validate()
#7 [internal function]: verbb\formie\controllers\SubmissionsController->actionSubmit()
#8 /Users/ckennedy/Code/manetu/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#9 /Users/ckennedy/Code/manetu/vendor/yiisoft/yii2/base/Controller.php(181): yii\base\InlineAction->runWithParams(Array)
#10 /Users/ckennedy/Code/manetu/vendor/craftcms/cms/src/web/Controller.php(190): yii\base\Controller->runAction('submit', Array)
#11 /Users/ckennedy/Code/manetu/vendor/yiisoft/yii2/base/Module.php(534): craft\web\Controller->runAction('submit', Array)
#12 /Users/ckennedy/Code/manetu/vendor/craftcms/cms/src/web/Application.php(278): yii\base\Module->runAction('formie/submissi...', Array)
#13 /Users/ckennedy/Code/manetu/vendor/craftcms/cms/src/web/Application.php(581): craft\web\Application->runAction('formie/submissi...', Array)
#14 /Users/ckennedy/Code/manetu/vendor/craftcms/cms/src/web/Application.php(257): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#15 /Users/ckennedy/Code/manetu/vendor/yiisoft/yii2/base/Application.php(392): craft\web\Application->handleRequest(Object(craft\web\Request))
#16 /Users/ckennedy/Code/manetu/web/index.php(26): yii\base\Application->run()
#17 /Users/ckennedy/.composer/vendor/laravel/valet/server.php(219): require('/Users/ckennedy...')
#18 {main}
```

**Steps to reproduce**
1. Set up a form with a phone number field.
2. Set that field to required.
3. Add a dropdown field with multiple choices
4. Set the phone number field to conditionally display only on one (or in my case two) of those dropdown choices.
5. Confirm that that conditional is set to "Show if Any"
6. Set the form to submit via AJAX.
6. Try to submit the form and see the 500 error.

### Form settings
- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.5
- Craft version: 3.6.16 (Solo)
- Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.5",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.6](https://github.com/verbb/formie/releases/tag/1.4.6)
